packages/web-components/src/components/ic-popover-menu/ic-popover-menu.tsx

```diff
@@ -131,7 +132,7 @@ export class PopoverMenu {
     }>
   ): void {
     if (!ev.detail.hasSubMenu && ev.detail.label !== "Back") {
-      this.closeMenu();
+      this.closeMenu(false, ev.detail.label);
```

Comment:
Suggested by @GCHQ-Developer-530 on internal channel: it might be good if the whole menu item was added to the event so we cover everything. Maybe updating `handleMenuItemClick` to emit the menu item element and then using that in the `icPopoverClosed` event